site/models/reference-quicklink.php

```diff
@@ -48,12 +47,9 @@ public static function childrenFromContentField(Field $field): array
 		return $children;
 	}
 
-	/**
-	 * Returns whether the referenced page is open
-	 */
-	public function isOpen(): bool
```

Comment:
`Issue:` Something seems to have been lost here. E.g. https://getkirby.test/docs/reference/objects/filesystem/dir/is-readable - on `main` "Dir" in the "Tools" section in the Reference sidebar is highlighted. With this PR it isn't anymore. Since `->isOpen()` is still used in some parts of the sidebar, we probably cannot remove this overwritten method here.